package.json

```diff
@@ -2127,6 +2127,248 @@
+													},
+													{
+														"key": "s",
+														"name": "Search",
+														"icon": "search",
+														"type": "command",
+														"command": "extension.coq.query.prompt.search"
+													}
+												]
+											},
+											{
+												"key": "p",
+												"name": "Send command to prover",
+												"icon": "console",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "f",
+														"name": "Finish coq computations",
+														"icon": "notebook-state-success",
+														"type": "command",
+														"command": "extension.coq.finishComputations"
+													},
+													{
+														"key": "i",
+														"name": "Interrupt coqtop backend",
+														"icon": "notebook-stop",
+														"type": "command",
+														"command": "extension.coq.interrupt"
+													},
+													{
+														"key": "q",
+														"name": "Quit coqtop backend",
+														"icon": "panel-close",
+														"type": "command",
+														"command": "extension.coq.quit"
+													},
+													{
+														"key": "r",
+														"name": "Reset coqtop backend",
+														"icon": "notebook-delete-cell",
+														"type": "command",
+														"command": "extension.coq.reset"
+													}
+												]
+											},
+											{
+												"key": "q",
+												"name": "Query prover about foucsed symbol",
+												"icon": "info",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "a",
+														"name": "About",
+														"icon": "info",
+														"type": "command",
+														"command": "extension.coq.query.about"
+													},
+													{
+														"key": "c",
+														"name": "Check",
+														"icon": "check",
+														"type": "command",
+														"command": "extension.coq.query.check"
+													},
+													{
+														"key": "l",
+														"name": "Locate",
+														"icon": "location",
+														"type": "command",
+														"command": "extension.coq.query.locate"
+													},
+													{
+														"key": "p",
+														"name": "Print",
+														"icon": "eye",
+														"type": "command",
+														"command": "extension.coq.query.print"
+													},
+													{
+														"key": "s",
+														"name": "Search",
+														"icon": "search",
+														"type": "command",
+														"command": "extension.coq.query.search"
+													}
+												]
+											},
+											{
+												"key": "s",
+												"name": "Set display options",
+												"icon": "gear",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "b",
+														"name": "Display all basic low level contents on/off",
```

Comment:
These bindings sound better to named "Toggle display of all basic low level contents"